### PR TITLE
Improves design for user settings page

### LINKED ISSUE #1575: New Design: Profile settings
## Overview

Change design for profile settings

UI_profile_settings

## Acceptance criteria
- [ ] Make it fit to our design
- [ ] Change design for tabs
- [ ] Adjust fonts size
- [ ] Adjust colors
- [ ] Change design of section headers
- [ ] Reduce spaces

### PR DESCRIPTION
**Pull request by Iloveall**
## Description

Fixes #1575

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

New tickets:
https://github.com/France-ioi/AlgoreaFrontend/issues/1590
https://github.com/France-ioi/AlgoreaFrontend/issues/1591

## Test cases

- [ ] Case 1:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/new-design/profile-settings/en/groups/users/670968966872011405/settings)
  3. Then I see new tabs design
  4. And I see design changes of content

- [ ] Case 2:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/new-design/profile-settings/en/groups/users/879359034123878481/settings)
  3. Then I see the message "You cannot access this page for this user"

- [ ] Case 3:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/new-design/profile-personal-data/en/groups/mine)
  3. Then I turn off network in dev panel
  4. And I click on Profile in header menu
  5. Then I see error state

### CODE REVIEW

File: src/app/groups/containers/user/user.component.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <ng-container *ngIf="state$ | async as state">
 
-  <alg-loading *ngIf="state.isFetching"></alg-loading>
+  <alg-loading class="alg-flex-1" *ngIf="state.isFetching"></alg-loading>
```
Comment (smadbe) [inline src/app/groups/containers/user/user.component.html:3]: This does not seem to work, the page is blank without loader while it is waiting. 

https://github.com/France-ioi/AlgoreaFrontend/assets/1053150/08df321c-a97a-4d7b-b4b6-02f46e2a428e

File: src/app/groups/containers/platform-settings/platform-settings.component.html [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
-              [defaultLang]="currentUser.defaultLanguage"
-              [redirectOnChange]="false"
-              (changeLang)="onChangeLang($event)"
-            ></alg-language-picker>
-          </span>
-        </li>
-      </ul>
-    </alg-section>
-  </div>
+  <h3 class="alg-h2 alg-text-normal title" i18n>Platform settings</h3>
```
Comment (smadbe) [inline src/app/groups/containers/platform-settings/platform-settings.component.html:2]: The title is not needed anymore.

**Review by smadbe — CHANGES_REQUESTED** [review]

File: src/app/groups/containers/user/user.component.html (hunk 1 above)
Comment (Iloveall) [reply src/app/groups/containers/user/user.component.html:3]: Fixed

File: src/app/groups/containers/platform-settings/platform-settings.component.html (hunk 2 above)
Comment (Iloveall) [reply src/app/groups/containers/platform-settings/platform-settings.component.html]: Fixed

**Review by smadbe — APPROVED** [review]
👍

**Pull request merged**